lambdas/process_webhooks/process_webhooks.py

```diff
@@ -169,37 +128,35 @@ def lambda_handler(event, _context):
     # Get the state of the spigot from the api variable
     spigot_state = event.get('spigot_state')
     logger.info(
-        "spigot_state is set to: {}'.format(spigot_state)"
+        "spigot_state is set to: {}".format(spigot_state)
     )
 
     if spigot_state == "ON":
-        # Get the url(s) that the webhook will be sent to
-        urls = _get_target_urls()
-        results = []
+        # Get the url that the webhook will be sent to
+        url = _get_target_url()
 
         # We had stored the payload to send in the
         # 'body' node of the data object.
         payload = event.get('body')
         logger.debug("payload is: '{}'".format(payload))
 
         # Send it off!
-        # Save up the results for later processing rather than letting
-        # the errors get raised.
-        # That way we can process all records and urls.
-        for url in urls:
-            results.append(_send_message(url, payload, headers))
-
-        results_count = _process_results_for_failures(results)
-
-        if results_count.get('failure'):
+        try:
+            _result = _send_message(url, payload, headers)
+        except:
             if not from_queue:
                 # The transmission was a failure, if it's not
                 # already in the queue, add it.
                 queue_name = _get_target_queue()
                 _response = _send_to_queue(event, queue_name)
-            raise StandardError(results_count)
+            raise StandardError(
+                "There was an error sending the message "
+                "to the target url."
```

Comment:
nit: since you know what the URL attempted was, include it in the output for easier troubleshooting.